pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
@mfojtik @rhcarvalho I'm as guilty as anyone about getting emotionally caught up in these sort of discussions, but as the (mostly) outside observer in this case:  I think it's time to take a step back and 1) take a deep breath, and 2) resume the discussion in person because this isn't the best/healthiest forum for a variety of reasons.

also one attempt at humor:  if this is our most contentious/important concern to spend our time on, then we have an amazing product.